ci/gpu/build.sh

```diff
@@ -203,7 +203,7 @@ else
     CONDA_FILE=`basename "$CONDA_FILE" .tar.bz2` #get filename without extension
     CONDA_FILE=${CONDA_FILE//-/=} #convert to conda install
     gpuci_logger "Installing $CONDA_FILE"
-    gpuci_mamba_retry install -c ${CONDA_ARTIFACT_PATH} "$CONDA_FILE"
+    gpuci_mamba_retry install -c rapidsai-nightly/label/testing -c ${CONDA_ARTIFACT_PATH} "$CONDA_FILE"
```

Comment:
Were these label changes added as a temporary change to test an unreleased `cumlprims` package? I'm not sure we want to commit this.

Reply:
Yes they were